ui/apps/platform/src/Containers/User/UserPage.jsx

```diff
@@ -85,43 +61,66 @@ function UserPage({ resourceToAccessByRole, userData }) {
                     </DescriptionListGroup>
                 </DescriptionListCompact>
             </PageSection>
-            <PageSection hasBodyWrapper={false} style={stylePageSection} isFilled>
-                <Flex>
+            <PageSection hasBodyWrapper={false} isFilled>
+                <Flex direction={{ default: 'column' }} gap={{ default: 'gapMd' }}>
                     <FlexItem>
-                        <div className="pf-v6-u-background-color-200">
-                            <Nav aria-label="Roles">
-                                <NavList>
-                                    <NavItem isActive={isUserPathActive}>
-                                        <NavLink to={userBasePath} end>
+                        <Flex direction={{ default: 'column' }} gap={{ default: 'gapMd' }}>
+                            <FlexItem>
+                                <SelectSingle
+                                    id="user-role-selector"
+                                    value={selectedRole}
+                                    handleSelect={handleRoleSelect}
+                                    placeholderText="Select a view"
+                                    isFullWidth={false}
+                                >
+                                    {[
+                                        <SelectOption
+                                            key="ALL"
+                                            value="ALL"
```

Comment:
Nit: it would be better to use a client side string for "ALL" that is impossible to overlap with one returned from the API.